standards/code-style.adoc

```diff
@@ -338,14 +342,59 @@ if (condition) {
 }
 ----
 
-This is __bad__:
-
 [source, kotlin]
 ----
 if (condition)
     body()  // bad!
 ----
 
+Braces are not required for `when` branches and `if` statement bodies if they meet ALL
+the following conditions:
+
+. there are no `else if`/`else` branches
+. the conditional and the branch contain at most a single predicate
+. it fits on a single line
```

Comment:
I would also argue that it's OK to omit braces for single-line \`when\` statements that include an else branch:
````kotlin
when (foo) {
    BAR -> doBar()
    BAZ -> doBaz()
    else -> return
}
````
(i.e. bullet 1 shouldn't apply to \`when\` statements)